tests/BusinessLogic/Common/TestComponents/Order/TestOrderRepository.php

```diff
@@ -94,7 +130,7 @@ public function getIncompleteOrderReferences()
      */
     public function getOrderAndShippingData($orderId)
     {
-        if ($this->throw) {
+        if ($this->throwOrderNotFoundException) {
```

Comment:
If possible, we'd like to have the orderId in the exception, so in the case of failure it would be easier to debug. Something like 
```throw new OrderNotFound('Order not found: ' . $orderId)```

Is it possible?

Reply:
(this comment applies to everywhere you have this "throw new OrderNotFound")

Reply:
This is a test and this is needed just for mocking results in order to test different scenarios. Order Id or shipment reference is always included in exceptions where thrown - and this is done in the modules.

Reply:
🆗